docs/parent/pom.xml

```diff
@@ -41,6 +41,8 @@
         <html.rel.file.prefix></html.rel.file.prefix>
         <mq.docs.url>https://eclipse-ee4j.github.io/openmq/guides/</mq.docs.url>
         <bookDirectory>${project.build.directory}/book</bookDirectory>
+        <productMajorVersion>${product.majorVersion}</productMajorVersion>
```

Comment:
Remove this property, and in adoc refer to {product-majorVersion}. This property is not set properly, because the build-helper plugin has not run yet.